### ISSUE TITLE
Shift all personal variables to secrets.php

### ISSUE THREAD

**Issue by wienbarg** [SenorSmartyPants/jellyfin-nmt#11]
At the top of utils.php there are two important variables
```
//NMT player path
$NMT_path = "/home/pi/Netzwerk/"; //server based path to share to NMT
$NMT_playerpath = "file:///opt/sybhttpd/localhost.drives/HARD_DISK/Netzwerk/";  //NMT path to the share
```
I'm wondering could secrets.php a better place? This way all personal variables would be in one place and new versions (which are coming **woohoo**) can easier be tested.

**Comment by SenorSmartyPants** [comment]
Missed that one. Thanks

Pushed fix QuickFixFilter branch in 9f4895db08b625ed397cc4041b35410799b465b8

README on website when I merge that into main.

**Comment by SenorSmartyPants** [comment]
Just move those lines to your secrets.php
